service-tests/github.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'github', title: 'GitHub' });
+module.exports = t;
+
+t.create('File size')
+  .get('/size/webcaetano/craft/build/craft.min.js.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('size'),
+    value: Joi.string().regex(/^[0-9]*[.]?[0-9]+\s(B|kB|MB|GB|TB|PB|EB|ZB|YB)$/),
+  }));
+
+t.create('File size 404')
+  .get('/size/webcaetano/craft/build/does-not-exist.min.js.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('size'),
+    value: Joi.string().regex(/^repo or file not found$/),
+  }));
+
+t.create('File size unknown')
+  .get('/size/webcaetano/craft/build.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('size'),
+    value: Joi.string().regex(/^unknown file$/),
+  }));
+
+t.create('downloads for release without slash')
```

Comment:
Ah, you can just remove the lines and commit the change.

Everything will get squashed, so the individual commits will go away. Probably this will go in first, and then #756 will need to solve a minor merge conflict (a new file created in both branches).